docs/python-stdlib-imports.md

```
@@ -0,0 +1,136 @@
+In summary, from a runtime perspective, bundling doesn’t fundamentally change how Python imports modules – it just means you have to package your code in a way that Python’s import system can still find the “modules.” Whether that’s by merging into one namespace or by cleverly intercepting import requests, the behaviors of `sys.modules` and import caching still apply. Once a module (standard library or your own) is loaded, it will sit in `sys.modules` and further imports of that name will reuse the cached module.
+
+## 6. Special Case: `from __future__` Imports
+
+- **Purpose**\
+  A `from __future__ import feature_name` statement tells the Python compiler to enable a feature that will become standard in a future release. In other words, it lets you use new syntax or semantics **before** it is the default. For example, in Python 3.9 you can write:
+  ```python
+  from __future__ import annotations
+  ```
+
+so that all function annotations are stored as strings rather than evaluated at definition time (PEP 563).
+•	Special-casing in the compiler
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 2

(MD010, no-hard-tabs)